code/datums/elements/art.dm

```diff
@@ -46,12 +46,11 @@
 	to_chat(user, span_notice("You start appraising [source]..."))
 	if(!do_after(user, 2 SECONDS, target = source))
 		return
-	var/mult = 1
+	var/mult
 	if(isobj(source))
 		var/obj/art_piece = source
-		mult = art_piece.get_integrity() / art_piece.max_integrity
-
-	apply_moodlet(source, user, impressiveness * mult)
+		art_piece.uses_integrity? (mult = art_piece.get_integrity() / art_piece.max_integrity) : 1
```

Comment:
```suggestion
		art_piece.uses_integrity ? (mult = art_piece.get_integrity() / art_piece.max_integrity) : 1
```
Why do we want to check integrity?

Reply:
Question for @Fikou too, why do we want to check something for integrity? It's such a weird system to base art around